titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobAssertions.java

```diff
@@ -30,12 +31,18 @@
 import com.netflix.titus.api.jobmanager.model.job.ContainerResources;
 import com.netflix.titus.api.jobmanager.model.job.Image;
 import com.netflix.titus.api.model.ResourceDimension;
+import com.netflix.titus.common.util.CollectionsExt;
 import com.netflix.titus.common.util.StringExt;
 
 /**
  */
 public class JobAssertions {
 
+    private static final Charset UTF_8 = Charset.forName("UTF-8");
+
+    public static final int MAX_ENVIRONMENT_SIZE_MB = 32;
```

Comment:
MB? Megabyte?

Reply:
KB?

Reply:
In the JIRA ticket, there is 64MB. We were talking about 32K yesterday, but I thought I misunderstood you.